cli/template/extras/config/next-config/with-auth-appdir.mjs

```diff
@@ -0,0 +1,20 @@
+/**
+ * Run `build` or `dev` with `SKIP_ENV_VALIDATION` to skip env validation. This is especially useful
+ * for Docker builds.
+ */
+import { env } from "./src/env.mjs";
+
+// Warn if local development URL doesn't match with the active port
+if (
+  process.env.NODE_ENV === "development" &&
+  !env.NEXTAUTH_URL.includes(String(env.PORT))
+) {
+  console.warn(
+    `NEXTAUTH_URL (${env.NEXTAUTH_URL}) doesn't match with PORT (${env.PORT})`
+  );
```

Comment:
is logging the right thing or should it be a hard throw?

Reply:
imo throw. logs are easy to miss, and if someone really wants to develop with a port that doesn't match their nextauth url (can't imagine this) they can always turn it off.